### DISCUSSION TITLE
离线安装 1panel

### DISCUSSION THREAD

**Discussion by yfdoors** [1Panel-dev/1Panel#4893] [Ideas]
### 1Panel 版本

lastest

### 问题描述

我按照这个教程 https://bbs.fit2cloud.com/t/topic/386 可以把 1panel 成功的安装到服务器上了，但现在依旧有两个问题：
1、文件管理中有的路径无法访问(例如 /home/user),提示没有权限。
2、应用安装：以 openresty 为例，把其他可以访问外网主机上 /opt/1panel/resource/apps/remote 中的文件拷到离线设备中的 “/opt/1panel/resource/apps/local”下，并把对应的 docker image 导出并导入到离线主机上。此时可以正常通过应用商店安装 openresty ， docker 可以正常创建，但是在 ”网站“ 标签也仍然提示未安装，无法管理网站。。

### 重现步骤

na

**Comment by hongbo-code** [comment]
同问
